package.json

```diff
@@ -38,8 +39,6 @@
     "truffle": "~5.8.2",
     "truffle-assertions": "~0.9.2",
     "truffle-flattener": "~1.6.0",
-    "truffle-plugin-verify": "~0.6.2",
-    "witnet-solidity": "^0.0.42",
-    "witnet-solidity-bridge": "https://github.com/guidiaz/witnet-solidity-bridge#460ba32fb6853e53f87c344f9f5263a93ac438ab"
```

Comment:
Why don't we need witnet-solidity-bridge if we are importing it in the code?

Reply:
it's now imported within the latest witnet-solidity package distribution.